website/docs/intro.md

```diff
@@ -2,7 +2,18 @@
 sidebar_position: 1
 ---
 
-# Introduction
-
-> Work in progress...
+# General
 
+> About plugin
+What is ECInventory for? It is a powerful tool for changing your inventory to your liking. With it, you can make your server even more atmospheric. You will be able to add a lot of new highly customizable inventories. Thanks to the Mimic API, ECInventory can use the capabilities of other plugins (MMOItems, MMOCore, QuantumRPG, SkillAPI, PROSkillAPI, Heroes, etc.).
+---
+# Where to begin? And what functionality does ECInventory contribute?
+> - Allows create more than one inventory
+> - Inventories are stored in SQL database
+> - Inventories support almost all possible clicks including Shift + Click, hotbar swap and offhand swap.
+> - Binding different commands to left, right, shift left and shift right clicks
+> - Define allowed and denied items from others plugins (via Mimic)
+> - Define allowed and denied items ids using wildcard patterns, for example pattern *_sword allows to put any sword to the slot.
+> - Placeholders (via PlaceholdersAPI) can be used in inventory names as well as in slot names, descriptions and commands.
+> - Admins can open other player's inventories and modify its content
```

Comment:
Symbol `>` is used for quote, so I don't think it is needed here. Also, let's change the list slightly:

```suggestion
- Create multiple inventories for players
- Inventories are stored in SQL database
- Inventories support almost all possible clicks including Shift + Click, hotbar swap and offhand swap
- Binding different commands to left, right, shift-left and shift-right clicks
- Define allowed and denied items from others plugins (via Mimic)
- Define allowed and denied items IDs using wildcard patterns, for example pattern `*_sword` allows to put any sword to the slot
- Placeholders (via PlaceholdersAPI) can be used in inventory names as well as in slot names, descriptions and commands
- Admins can open other player's inventories and modify its content
```